puma/plot_base.py

```diff
@@ -3,6 +3,7 @@
 from dataclasses import dataclass
 
 import atlasify
+import matplotlib as mpl
```

Comment:
why are you importing `mpl` here?
as far as I can it you could add in the line below `lines` and it would work?

Reply:
true, I'll change that